yarn.lock

```diff
@@ -11,13 +11,6 @@
     ethereumjs-util "^6.1.0"
     web3 "^1.0.0-beta.55"
 
```

Comment:
should tokens@0.1.2 be here as well?

Reply:
no it's removing it from the lock file. It doesn't need it in `yarn.lock` as it can get version 0.1.2 from the repository itself. It only pulls in airswap modules into here when its not the current version

Reply:
just noticed the import of order utils above this. That means the repo is using an old version somewhere too

Reply:
https://github.com/airswap/airswap-protocols/pull/269/commits/adb413621580f273871414424a67ab7fa1ab3b5e